src/test/java/com/epam/tc/hw3/test/TestsMainPage.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.tc.hw3.test;
+
+import com.epam.tc.hw3.pages.HeaderMenuPage;
+import com.epam.tc.hw3.pages.LeftMenuPage;
+import com.epam.tc.hw3.pages.MainPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.annotations.Test;
+
+public class TestsMainPage extends AbstractTest {
+
+    MainPage mainPage;
+    HeaderMenuPage headerMenuPage;
+    LeftMenuPage leftMenuPage;
+    private final String scrollFooter = "window.scrollBy(0,500)";
+    private final String scrollHeader = "window.scrollBy(0,-500)";
```

Comment:
Please, move all the manipulations with pages to page layer, don't keep it in test layer